lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/DefaultRemoteManagerFactory.java

```diff
@@ -69,7 +66,7 @@ private DefaultRemoteManagerFactory(
   @Override
   public RemoteManager getInstance(final String name) {
     try {
-      final Injector newInjector = injector.forkInjector();
+      final Injector newInjector = Tang.Factory.getTang().newInjector();
```

Comment:
You're right. I changed the code according to your advice.
Since the current master keep that instance variable, it reduce the size of this PR, too.
Thank you, @markusweimer .